types/node/globals.d.ts

```diff
@@ -664,7 +664,7 @@ declare namespace NodeJS {
     type ExitListener = (code: number) => void;
     type RejectionHandledListener = (promise: Promise<any>) => void;
     type UncaughtExceptionListener = (error: Error) => void;
-    type UnhandledRejectionListener = (reason: Error | any, promise: Promise<any>) => void;
+    type UnhandledRejectionListener = (reason: Error | object, promise: Promise<any>) => void;
```

Comment:
That's wrong now as you can `throw 15` or `throw undefined`.
```suggestion
    type UnhandledRejectionListener = (reason: unknown, promise: Promise<unknown>) => void;
```

Reply:
Hmm.. looks interesting but this still does not solve my case (

Reply:
``` 
process.on('unhandledRejection', (reason: unknown) => {
  let text = reason;
  let extra: {} = {};
  if (reason != null && (reason as any).message) {
    const { message, ...rest } = reason as any;
    text = message;
    extra = rest;
  }
  logger.error(`### Unhandled rejection: ${text}`, extra);
});
```
Feel free to us a more specific type then `any` above. You could consider also that `reason.message` could be a `symbol` which would once again cause a crash.

Reply:
i checked this case with unknown and it works. thanks. 
well, unknown is  ok

```typescript
  if (reason instanceof Error && reason.message) {
    const { message, ...rest } = reason;
    text = message;
    extra = rest;
  }
```